### PR TITLE
Update camera manager to t3.3.0

### PR DESCRIPTION
**Pull request by patrickelectric**
- Add thermal option

docker: patrickelectric/bootstrap:camera-manager-3-3-0

image

### CODE REVIEW

**Comment by rafaellehmkuhl** [comment]
Are we planning to add much more options on that "extra" menu? If not, I would suggest putting the checkbox on the same dialog, without the extra expanding menu, like we have with the endpoints.

image

If we plan to put much more extra options there, I would just put this whole menu at the end (after stream endpoint), as its extra.

**Comment by patrickelectric** [comment]
> Are we planning to add much more options on that "extra" menu? If not, I would suggest putting the checkbox on the same dialog, without the extra expanding menu, like we have with the endpoints.
> 
> image
> 
> If we plan to put much more extra options there, I would just put this whole menu at the end (after stream endpoint), as its extra.

Yes, we are going to add more later with future versions.

File: core/frontend/src/components/video-manager/VideoStream.vue [hunk 1]
```diff
@@ -104,6 +104,7 @@ export default Vue.extend({
         },
         interval: this.stream.video_and_stream.stream_information.configuration.frame_interval,
         endpoint: this.stream.video_and_stream.stream_information.endpoints[0],
+        thermal: this.stream.video_and_stream.stream_information?.extended_configuration?.thermal ?? false,
```
Comment (rafaellehmkuhl) [inline core/frontend/src/components/video-manager/VideoStream.vue:107]: Cool double-question-mark operator. Didn't know.

**Review by rafaellehmkuhl — APPROVED** [review]

**Comment by rafaellehmkuhl** [comment]
> > Are we planning to add much more options on that "extra" menu? If not, I would suggest putting the checkbox on the same dialog, without the extra expanding menu, like we have with the endpoints.
> > image
> > If we plan to put much more extra options there, I would just put this whole menu at the end (after stream endpoint), as its extra.
> 
> Yes, we are going to add more later with future versions.

Ok. In this case, I would suggest just putting it at the end.

**Comment by patrickelectric** [comment]
image

**Review by Williangalvani — APPROVED** [review]

**Pull request merged**